product.html

```diff
@@ -127,8 +127,8 @@
         </div>
     </section>
 
-    <section class="insights">
-        <div class="grid grid-cols-[45%,1fr] items-center gap-20 p-20 relative z-10">
+    <section class="insights">grid grid-cols-[45%,1fr] items-center gap-20 p-20 relative z-10
```

Comment:
These classes shouldn't be here. Consider moving them to the external CSS file or simply delete them if you're not using them and already have `insights` block.